README.md

```diff
@@ -1 +1,15 @@
-# PW_train_YAJSC
\ No newline at end of file
+# PW_train_YAJSC
+
+## Requirements
+
+- NodeJS
+- VSCode
```

Comment:
other tools such as AQA or WebStorm could be used as well (node js Integrated development environment Software)